README.md

```diff
@@ -5,7 +5,7 @@ This template provides a minimal setup to get React working in Vite with HMR and
 Currently, two official plugins are available:
 
 
-# TODO: Please add high-quality instructions that each of your group members can use to run this project in dev mode. Please include instructions for telling members to git pull, npm install, and then the commands for runnnig your project
+# TODO: Please add high-quality instructions that each of your group members can use to run this project in dev mode. Please include instructions for telling members to git pull, npm install, and then the commands for running your project
```

Comment:
Please add a readme with steps for running the project, so each group member feels confident running things.